set.mm

```diff
@@ -50686,12 +50726,35 @@ under a function is also a set (see the variant ~ funimaex ).  Although
   $}
 
   ${
-    $d x ph z $.  $d x y z $.
+    $d x y z $.  $d x y ph $.
+    bm1.3iiv.1 $e |- E. x A. z ( ph -> z e. x ) $.
+    $( Version of ~ bm1.3ii combined with a change of variable, requiring fewer
+       axioms.  (Contributed by Matthew House, 18-Sep-2025.) $)
+    bm1.3iiv $p |- E. y A. z ( z e. y <-> ph ) $=
```

Comment:
bm1.3ii should be provable with the same axioms as bm1.3iiv:

```
E. x A. z ( ph -> z e. x )

bm1.3iiv $d x y z $d x y ph
E. y A. z ( z e. y <-> ph )

eximii(^, >) albii biimpr
E. y A. z ( ph -> z e. y )

bm1.3iiv $d y x z $d y x ph
E. x A. z ( z e. y <-> ph )
```

this might re-allow some bm1.3ii uses

Reply:
I decided to go and add the explicit closed form anyway, just because inference form is wacky, and I may find some use for this conditional ax-sep in future quixotic "ZFC sans ax-8 & ax-9" adventures (most of the results of which are likely better left to a mathbox).